lib/src/client/call.dart

```diff
@@ -204,6 +226,9 @@ class ClientCall<Q, R> implements Response {
 
   /// Emit an error response to the user, and tear down this call.
   void _responseError(GrpcError error) {
+    _timeline?.finish(arguments: {
```

Comment:
We build this object a lot. Let's make a helper for this, something like:

```dart
void _writeErrorToTimeline(TimelineTask timeline, GrpcError error) {
  timeline?.finish(arguments: {
    'error': error.toString(),
  );
}
```

Reply:
Done